src/wizard/wizard-form.tsx

```diff
@@ -43,6 +44,14 @@ export default function WizardForm({
   const isLastStep = activeStepIndex >= steps.length - 1;
   const skipToTargetIndex = findSkipToTargetIndex(steps, activeStepIndex);
   const isMobile = useMobile();
+  const stepHeaderRef = useRef<HTMLDivElement | null>(null);
+
+  useEffectOnUpdate(() => {
+    if (stepHeaderRef && stepHeaderRef.current) {
+      const header = stepHeaderRef.current.querySelector('[tabindex = "-1"]') as HTMLElement;
```

Comment:
Can we avoid `querySelector` and expose the correct element to focus right away?